enhancements/cloud-integration/tokenized-auth-enablement-operators-on-cloud.md

```diff
@@ -169,7 +169,16 @@ CCO should consider adding the above, three-part test as a utility function expo
 **HyperShift changes**: Include Cloud Credential Operator with token-aware mode (see above). Allows for processing of 
 CredentialsRequest objects added by operators. 
 
-The CCO is expected to run on the control plane NS, not on the customer's worker nodes, and not be visible to the customer. However if CredentialsRequests are created on the worker nodes, the CCO will process them and yield Secrets as described elsewhere in this document.
+The CCO is expected to run on the control plane NS, not on the customer's worker nodes, and not be visible to the customer. We propose that multiple 
+instances of CCO be installed on the control plane each consuming the 
+kubeconfig of the worker cluster they are intended to be watching / managing. 
+The additional cost of adding CCO to Hypershift is assumed by Red Hat and not 
+by the worker nodes.
+
+CCO will be modified so that it can consume a given kubeconfig when specified. 
+Because the intention is to make this feature available with HyperShift GA it 
+will need to be backported all the way to 4.12 (contingent on CCO changes 
+moving out of the feature gate).
```

Comment:
not sure this backport is necessary(SD has not explicitly required it).  At a minimum it's contingent on us backporting the rest of the STS-enablement work (e.g. console) to 4.12.

if we didn't backport it, it would just mean that you only get this behavior on 4.14+ guest clusters and not 4.12/4.13 guest clusters.

it also might mean the hypershift operator would need to understand the version difference, depending on whether it needs to do anything special to provide the hosted cluster's kubeconfig to the CCO.